### ISSUE TITLE
Ping Failed but packet received.

### ISSUE THREAD

**Issue by faizolsaidin** [seunomosowon/TA-connectivity#4]
![image](https://user-images.githubusercontent.com/33227405/38379797-6df1b62e-3934-11e8-86b7-a00fb4b4431c.png)

**Comment by faizolsaidin** [comment]
Just started on TA-Con, am i missing any setting or conf?

**Comment by seunomosowon** [comment]
No, I don’t think you are. Must be a bug when I moved the old code over.
I’ll look into it over the weekend and let you know once it’s fixed.

**Comment by faizolsaidin** [comment]
Primarily, i had a IP Printer that inside Windows Printer Server and would like to use TA-Con to monitor the printer avail. Based from pinger.py, it only decide on 2 platform, either windows or linux. How does it goes for IP Printer, any opinion?

**Comment by faizolsaidin** [comment]
I found this in pinger.py

![image](https://user-images.githubusercontent.com/33227405/38477845-a48e3eb8-3be7-11e8-85b2-d8672b956d51.png)

It seems the ping was failed but able to get rtt value and it return as nix_errors since it could not be read as windows host.

**Comment by faizolsaidin** [comment]
Do you have any fix on this? Its neither Windows or Linux, its IP Printer, it is possible?

**Comment by seunomosowon** [comment]
Hi,

Are you running this on windows or linux?

It doesnt report correctly if you’re running this on a windows system. I suspect you have an error running it on windows. I’ll put a fix in this summer.

Best regards,
Oluwaseun Remi-Omosowon

**Comment by faizolsaidin** [comment]
Linux but the IP Printers were running in Windows Server. I believe so, it seems not able to identify the printer as devices. 

Appreciate your feedback once you had the fix.

**Comment by seunomosowon** [comment]
The section of the code you are looking at is only about the platform it is
running on. Check that you are able to ping it directly from your OS. It
doesn’t matter if the device is a printer.

Do you get any specific errors in the app?

**Comment by faizolsaidin** [comment]
Sorry for late reply

So far i don't find and see any error from the app. 

It able to ping from my OS. Does it have any criteria if needed since its not a system?

Faizol

**Comment by seunomosowon** [comment]
Are you running this on a Windows Or Linux system?

Best regards,
Oluwaseun Remi-Omosowon

**Comment by faizolsaidin** [comment]
Splunk Instance = Linux
Printer Platform = Windows

**Comment by faizolsaidin** [comment]
Hi,

Anyway if its due to the host that contain special char such as '-'?

![image](https://user-images.githubusercontent.com/33227405/46725333-8a367280-ccae-11e8-97bc-06ca96abcc83.png)

It had no issue with IP address, though.
